app/app.js

```diff
@@ -14,19 +14,20 @@
  * limitations under the License.
  */
 
-import Ember from 'ember';
-import Resolver from 'ember/resolver';
-import loadInitializers from 'ember/load-initializers';
-import config from './config/environment';
+// import Ember from 'ember';
+const Ember = require('ember');
+const Resolver = require('ember/resolver');
+const loadInitializers = require('ember/load-initializers');
+const config = require('./config/environment');
 
 Ember.MODEL_FACTORY_INJECTIONS = true;
 
-var App = Ember.Application.extend({
+const App = Ember.Application.extend({
   modulePrefix: config.modulePrefix,
   podModulePrefix: config.podModulePrefix,
-  Resolver: Resolver
+  Resolver
 });
 
 loadInitializers(App, config.modulePrefix);
 
-export default App;
+module.exports = App;
```

Comment:
Can you explain this change everywhere? I don't see any indication on the style guide that `export default` is to be avoided...

Reply:
Yes, you are right. Reverted changes.